src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -574,21 +575,41 @@ def handleSpecialCERNMergeSettings(self, funcName):
         Enable lazy-download for fastCloning for all CMSSW_7_5 jobs (currently off)
         Enable lazy-download for all merge jobs
         """
-        cmsswVersion = os.environ['CMSSW_VERSION']
+        cmsswVersion = os.environ.get('CMSSW_VERSION', "")
         if cmsswVersion.startswith("CMSSW_7_5") and False:
```

Comment:
It was Brians automatic "if job uses fastCloning then enable lazy download" feature which sounded great in theory, but wasn't actually working in practice (most likely because the check if a job was using fastCloning was too expensive and therefore it spend too much time on that before it had a chance to switch to lazy download).

I should probably remove this completely, although don't want to do this as part of this pull request.